frontend/src/components/Register/Input.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+import { tertiaryColor } from "../../utils/styleVariables";
+
+
+const Input = styled.input`
+  padding: .5rem;
+  border: none;
+  border-bottom: 1px solid ${tertiaryColor};
```

Comment:
What if you are gonna to change website theme for blue? 
variable called `tertiaryColor` will for example be blue?

Reply:
Then? PrimaryColor and SecoundaryColor is ok, so why TertiaryColor no?

Reply:
My bad, I thought ` Tertiary` is some kind of red turkey color